### PR TITLE
[documentation] #2193: Update Iroha CLI documentation

### PR DESCRIPTION
**Pull request by outoftardis**
Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

### Description of the Change

Updated readme: https://github.com/outoftardis/iroha/tree/doc-cli/cli#iroha-cli

### Issue

Partially addresses #2193

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -47,135 +42,125 @@ This flag can be combined with the `--features` flag in order to precisely speci
...
+:grey_exclamation: We do not recommend using environment variables for configuration outside docker-compose and Kubernetes deployments. Please change the values in the configuration file instead. That would also help us debug the problems that you might be having.
 
-A [sample configuration file](../configs/peer/config.json) is provided for quick testing.
+The [configuration options reference](../docs/source/references/config.md) provides detailed explanations of each configuration variable. You may use the [sample configuration file](../configs/peer/config.json) for quick testing.
 
 One of the peers on your network must be provided with the genesis block, which is either `IROHA2_GENESIS_PATH` or `genesis.json` in the working directory.
```
Comment (Arjentix) [inline cli/README.md:97]: I think we also need to describe what genesis is and what does `genesis.json` contain. Or leave some link to this description

**Review by Arjentix — APPROVED** [review]

File: cli/README.md [hunk 2]
```diff
@@ -1,43 +1,38 @@
 # Iroha CLI
 
-## Description
+The binary `iroha` crate contains the Iroha peer binary. The binary is used to instantiate a peer and bootstrap an Iroha-based network. The capabilities of the network are determined by the feature flags used to compile the binary.
```
Comment (appetrosyan) [inline cli/README.md:3]:
```suggestion
The `iroha` crate contains the Iroha peer binary, which is used to instantiate a peer and bootstrap an Iroha-based network. The capabilities of the network are determined by the feature flags used to compile the binary.
```

Comment (appetrosyan) [inline cli/README.md:3]:
```suggestion
The `iroha` crate contains the Iroha peer binary, which is used to instantiate a peer and bootstrap an Iroha-based network. The capabilities of the network are determined by the feature flags used to compile said binary.
```

**Review by appetrosyan — CHANGES_REQUESTED** [review]

File: cli/README.md (hunk 1 above)
Comment (outoftardis) [reply cli/README.md]: added links to the config options in docs and to the tutorial

**Review by appetrosyan — APPROVED** [review]

**Review by Arjentix — APPROVED** [review]

**Pull request merged**